testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -774,6 +797,11 @@ protected SELF self()
         public DistributedQueryRunner build()
                 throws Exception
         {
+            if (System.getenv("CONTINUOUS_INTEGRATION") == null) {
+                // For non-CI usage it is ok to enable tracing for all query runners
+                withTracingEnabled();
```

Comment:
This adds extra cost of starting up the collector container. It's image also needs to be downloaded on first use. Should this be opt-in for local development?

Reply:
Image is small and starts under 1s

Reply:
And how much memory does it require? I just wonder if all developers will benefit from this and is the extra resources cost justified.

Reply:
Sure.